tests/entrypoints/openai/test_translation_validation.py

```diff
@@ -79,7 +79,7 @@ async def test_basic_audio_with_lora(mary_had_lamb):
             temperature=0.0,
         )
     out = json.loads(translation)["text"].strip().lower()
-    assert "mary tenía un pequeño cordero" in out
+    assert "pequeño" in out
```

Comment:
Though I prefer to be slightly stricter

```suggestion
    assert "pequeño" in out.split(" ")
```

Reply:
Sounds good to me. Thanks for the quick review!